utils/InputFormat.tsx

```diff
@@ -0,0 +1,32 @@
+import {ethers} from 'ethers';
+
+// Checks whether the value of an input is a valid value.
+// Returns undefined if valid, or an error message string if invalid.
+export type InputFormat = (string) => string | undefined;
```

Comment:
not a big deal but tslint was showing a warning here `Parameter has a name but no type. Did you mean 'arg0: string'?ts(7051)`

Reply:
ah... yeah i gotta set up ci. let me do that next.